End-to-End-Tests.md

```diff
@@ -258,7 +258,7 @@ python -m scripts.run_e2e_tests --prod_env
 
 Docker:
 ```console
-make run_tests.e2e PYTHON_ARGS="--prod_env"
+make run_tests.e2e prod_env=true
```

Comment:
Please undo this change. Refer to https://github.com/oppia/oppia/wiki/Installing-Oppia-using-Docker#running-tests

Reply:
Done